### PR TITLE
Adds proxy for Provider mode backend in ConsolePlugin

### PR DESCRIPTION
**Pull request by bipuladh**
Adds proxy for console to connect to the token generation server. This is required for Provider mode on-boarding feature.

### CODE REVIEW

File: console/console.go [hunk 1]
```diff
@@ -94,6 +94,17 @@ func GetConsolePluginCR(consolePort int, serviceNamespace string) *consolev1alph
...
+				Type:  consolev1alpha1.ProxyTypeService,
+				Alias: "provider-proxy",
+				Service: consolev1alpha1.ConsolePluginProxyServiceConfig{
+					Name:      "ux-backend-proxy",
+					Namespace: serviceNamespace,
+					Port:      8888,
+				},
+				Authorize: true,
+			},
+			},
```
Comment (iamniting) [inline console/console.go:107]:
```suggestion
			Proxy: []consolev1alpha1.ConsolePluginProxy{
			    {
				Type:  consolev1alpha1.ProxyTypeService,
				Alias: "provider-proxy",
				Service: consolev1alpha1.ConsolePluginProxyServiceConfig{
					Name:      "ux-backend-proxy",
					Namespace: serviceNamespace,
					Port:      8888,
				},
				Authorize: true,
			    },
			},
```

**Review by iamniting — CHANGES_REQUESTED** [review]
pls fix the commit msg also

**Comment by rchikatw** [comment]
Also add description to the PR

**Review by iamniting — APPROVED** [review]

**Pull request merged**